Sources/SmithyHTTPAPI/SdkHttpRequest.swift

```diff
@@ -81,29 +81,30 @@ public final class SdkHttpRequest: RequestMessage {
     }
 }
 
-public extension URLRequest {
-    init(sdkRequest: SdkHttpRequest) async throws {
+public extension SdkHttpRequest {
+    static func makeURLRequest(from sdkRequest: SdkHttpRequest) async throws -> URLRequest {
```

Comment:
Changed this to an extension on `SdkHttpRequest` to prevent polluting `URLRequest` with it.